candidate-standard/abstract_tests/geojson/ATS_content.adoc

```diff
@@ -6,4 +6,5 @@
 ^|Requirement |<<req_geojson_content,/req/geojson/content>>
 ^|Test Method |. Validate that the document is a GeoJSON document.
 . Validate the document against the schema using an JSON Schema validator.
+. Validate the document against the schema using an GeoJSON Schema validator.
```

Comment:
Could we replace this with something more definitive,  using OpenAPI allows EDR to supply the JSON schemas for the data being served.  Rather than having a test for GeoJSON I suggest we replace it with a test that validates the data against the schemas defined in the OpenAPI definition.  This test could then be used for any JSON schema supported by the API and would help with interoperability.

Reply:
@m-burgoyne I agree that using the facilities of OpenAPI V3.0 is preferable, as more general, less effort, and more automatable. I am not expert enough to draft those lines, so I think it is a task for you over the next couple of weeks, and anyone else who can step up.

Reply:
@m-burgoyne Let's go with the branch PR as it is and then we make a new branch this afternoon to finish the full suite of tests.

Reply:
I don't think there is any difference between "GeoJSON Schema Validator" and "JSON Schema Validator" ... just sayin'

Reply:
@dblodgett-usgs There was no difference in the OGC API-Common-Core spec, they both just tested for JSON. I am not sure whether that was intentioanl or a typo. Either way, I think there is merit in having stricter tests for GeoJSON and CoverageJSON if there are schemas to do it. I'm applying "Postel's Law": be generous in what you accept, be parsimonious (strict) in what you give out.

Reply:
We also have an even more restrictive schema for `items` -- will that also be separate? 

Are we maybe missing a nuance here? There's JSON syntax validation then there JSON-schema validation. Since there are schema for both geojson and coverage json, I just don't see the difference other than the schema having a well-known name.